dvc/config_schema.py

```diff
@@ -224,10 +225,16 @@ class RelPath(str):
         "row_limit": All(Coerce(int), Range(1)),  # obsoleted
         "row_cleanup_quota": All(Coerce(int), Range(0, 100)),  # obsoleted
     },
+    "machine": {
+        str: {
+            "cloud": All(Lower, Choices("aws", "azure")),
```

Comment:
maybe

```suggestion
            "cloud": All(Lower, Choices("aws", "azure", "gcp", "kubernetes")),
```

/CC @iterative/cml

Reply:
See also https://github.com/iterative/dvc/blob/f77a5d2402600272eee6fff5de9cd21ca511c60a/dvc/machine/__init__.py#L91

Reply:
Do gcp and kubernetes work with the existing TPI? I was just going off of what's documented in the readme right now

> The Iterative Provider currently supports AWS and Azure. Google Cloud Platform is not currently supported.

Reply:
Either way, this kind of thing can also be addressed later (https://github.com/iterative/dvc/issues/6480)

This PR is not meant to be a complete/finished implementation in any way

Reply:
Yes `gcp` & `kubernetes` are in the same category as `dvclive` - they *should* work even though the docs aren't there yet.